log/log.go

```diff
@@ -0,0 +1,201 @@
+// Copyright © 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
+
+// Specifies the log levels
+const (
+	LevelError = iota + 1
+	LevelWarning
+	LevelInfo
+	LevelDebug
+	LevelVerbose // This is the same as Debug, but without the repeat filtering
+)
+
+// Specifies the command tags
+const (
+	Mixer       = "MIXER"
+	Dnf         = "DNF"
+	Rpm2Archive = "RPM2ARCHIVE"
+	Tar         = "TAR"
+	BsDiff      = "BSDIFF"
+	Mca         = "MCA"
+	Ssl         = "SSL"
+	Git         = "GIT"
```

Comment:
Can we have one for `createrepo_c`.
	`CreateRepo         = "CREATEREPO"`

and **not** write its logs to stdout. Instead write them as log.Debug
```
Directory walk started
Directory walk done - 0 packages
Temporary output repo path: ./.repodata/
Preparing sqlite DBs
Pool started (with 5 workers)
Pool finished
```